compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -1888,13 +1888,13 @@ impl EncodeContext<'a, 'tcx> {
                         default.is_some(),
                     );
                 }
-                GenericParamKind::Const { .. } => {
-                    self.encode_info_for_generic_param(
-                        def_id.to_def_id(),
-                        EntryKind::ConstParam,
-                        true,
-                    );
-                    // FIXME(const_generics_defaults)
+                GenericParamKind::Const { ref default, .. } => {
+                    let def_id = def_id.to_def_id();
+                    self.encode_info_for_generic_param(def_id, EntryKind::ConstParam, true);
+                    if default.is_some() {
+                        self.encode_stability(def_id);
```

Comment:
The call to `encode_stability` should not be needed. The correct case should already be handled by `should_encode_stability`.